services/user.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.hashers import make_password
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> User:
+
+    user = get_user_model().objects.create(username=username)
```

Comment:
`The method should save user properly (password should be encrypted), use method .create_user() for this purpose`
Read checklist again, please 🙌